tar/tar.c

```diff
@@ -2737,23 +2737,16 @@ checkzip(const char *bp, int rd)
 {
 	if (rd <= TBLOCK || (memcmp(&bp[257], "ustar", 5) &&
 			memcmp(&bp[257], "\0\0\0\0\0", 5))) {
-		if (bp[0] == 'B' && bp[1] == 'Z' && bp[2] == 'h')
+		iif (bp[0] == 'B' && bp[1] == 'Z' && bp[2] == 'h')
```

Comment:
Well, we have gotten a small syntax error here.
Nothing really special, the code itself seems right.